Misc/NEWS.d/next/Library/2020-09-26-12-16-28.bpo-41861.3xTsIp.rst

```diff
@@ -0,0 +1 @@
+Covert :mod:`sqlite3` Cache and Node types to heap types (PEP 384).
```

Comment:
Please remove the NEWS entry.

I would prefer to have a single entry added by the PR which will convert the last type, rather than having one entry per converted type.

Reply:
Of course.

Reply:
Fixed in f1568b1ea971dfbe04a8ccf9b6072ec116f47e7c